src/main/java/com/gtbr/vaultbot/model/PrivateChannel.java

```diff
@@ -0,0 +1,32 @@
+package com.gtbr.vaultbot.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@Table
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PrivateChannel {
+
+    @Id
+    @Column
+    private String privateChannelId;
+
+    @OneToOne
+    @JoinColumn(name = "guild_id")
+    private Guild guildId;
+
+    @OneToOne
+    @JoinColumn(name = "user_id")
+    private UserProfile userProfileId;
```

Comment:
this attribute should be named as "userProfile" because we will access it as an object